startup_scripts/stuff-org

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
If we have a systemd stuff-org.service, we probably shouldn't do this via the detour of a sysV init script that then is invoked by systemd, but rather have systemd start the service directly.